parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ParquetFileReader.java

```diff
@@ -519,7 +526,7 @@ public ParquetFileReader(
     this.conf = configuration;
     this.fileMetaData = fileMetaData;
     FileSystem fs = filePath.getFileSystem(configuration);
-    this.f = fs.open(filePath);
+    this.f = HadoopStreams.wrap(fs.open(filePath));
```

Comment:
I'm not sure we need both the stream and the file to be one object, but I'm not sure what the right design for it is. I think it's pretty safe to say we'll want a seekable stream and we can add a ScannableFile that inherits from that stream later when we design a Hadoop-free API.